dev-packages/e2e-tests/maestro/captureException.yml

```diff
@@ -1,5 +1,10 @@
 appId: ${APP_ID}
+jsEngine: graaljs
 ---
 - runFlow: utils/launchTestAppClear.yml
-- tapOn: "Capture Exception"
+- tapOn: 'Capture Exception'
 - runFlow: utils/assertEventIdVisible.yml
+- runFlow:
+    file: utils/assertReplay.yml
+    when:
+      platform: iOS
```

Comment:
I would create standalone replay on error test and enable replay thru launch args. 

To separate capturing errors and capturing replay on error.

Reply:
done

Reply:
bd6dc14fd31058432bd8ec794d8b0e76ec0a7dc6